lambdas/issuecredential/src/main/java/uk/gov/di/ipv/cri/passport/issuecredential/IssueCredentialHandler.java

```diff
@@ -72,15 +97,44 @@ public APIGatewayProxyResponseEvent handleRequest(
             PassportCheckDao passportCheck =
                     dcsPassportCheckService.getDcsPassportCheck(resourceId);
 
-            PassportCriResponse passportCriResponse =
-                    PassportCriResponse.fromPassportCheckDao(passportCheck);
+            VerifiableCredential verifiableCredential =
+                    VerifiableCredential.fromPassportCheckDao(passportCheck);
 
-            return ApiGatewayResponseGenerator.proxyJsonResponse(
-                    HttpStatus.SC_OK, passportCriResponse);
+            SignedJWT signedJWT = generateAndSignVerifiableCredentialJwt(verifiableCredential);
+
+            return ApiGatewayResponseGenerator.proxyJoseResponse(
```

Comment:
The `proxyJoseResponse` method will set the response content type to `application/jose`. We'll need to set it to `application/jwt` when returning the signed VC or core will blow up when calling `response.getContentAsJWT()`. It might be that you can just rename this method and change the content type it it's not used elsewhere. Otherwise you might need a new method.